pallets/automation-time/src/tests.rs

```diff
@@ -251,6 +251,35 @@ fn schedule_duplicates_errors() {
 	})
 }
 
+#[test]
+fn schedule_max_execution_times_errors() {
+	new_test_ext(START_BLOCK_TIME).execute_with(|| {
+		Balances::set_balance(RawOrigin::Root.into(), ALICE, 100_000, 5).unwrap();
+		assert_noop!(
+			AutomationTime::schedule_notify_task(
+				Origin::signed(ALICE),
+				vec![50],
+				vec![SCHEDULED_TIME, SCHEDULED_TIME + 120, SCHEDULED_TIME + 240, SCHEDULED_TIME + 360],
+				vec![2, 4]
+			),
+			Error::<Test>::TooManyExecutionsTimes,
+		);
+	})
+}
+
+#[test]
+fn schedule_execution_times_removes_dupes() {
+	new_test_ext(START_BLOCK_TIME).execute_with(|| {
+		Balances::set_balance(RawOrigin::Root.into(), ALICE, 100_000, 5).unwrap();
+		assert_ok!(AutomationTime::schedule_notify_task(
+			Origin::signed(ALICE),
+			vec![50],
+			vec![SCHEDULED_TIME, SCHEDULED_TIME, SCHEDULED_TIME, SCHEDULED_TIME, SCHEDULED_TIME + 360],
```

Comment:
prob worth pulling the scheduled task to make sure only 2 times are in the execution times vector (and that they are what you expect)